.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: true
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode9.4
+      language: node_js
+      node_js:
+        - 8
+    - os: linux
+      services: docker
+      language: node_js
+      node_js:
+        - 8
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - libudev-dev
+            - libusb-1.0-0-dev
+            - g++-4.8
+            - g++-4.8-multilib
+            - icnsutils
+            - graphicsmagick
+            - xz-utils
+            - xorriso
+            - rpm
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+install:
+  - yarn
+
+script:
+  - node --version
+  #- yarn lint
```

Comment:
Why is the linter turned off?

Reply:
There is no `lint` script, yet.
But I think it will be necessary after.